tests/AutoDiscoveryHelperTest.php

```diff
@@ -163,4 +163,26 @@ public function test_it_finds_lang_directories(): void
 		$this->assertContains($this->module1->path('resources/lang'), $resolved);
 		$this->assertContains($this->module2->path('resources/lang'), $resolved);
 	}
+
+	public function test_it_finds_livewire_component(): void
+	{
+		$this->artisan(MakeComponent::class, [
+			'name' => 'TestComponent',
+			'--module' => $this->module1->name,
+		]);
+
+		$this->artisan(MakeComponent::class, [
+			'name' => 'TestComponent',
+			'--module' => $this->module2->name,
+		]);
+
+		$resolved = [];
+
+		$this->helper->bladeComponentFileFinder()->each(function(SplFileInfo $file) use (&$resolved) {
+			$resolved[] = $file->getPathname();
+		});
+
+		$this->assertContains($this->module1->path('src/View/Components/TestComponent.php'), $resolved);
+		$this->assertContains($this->module2->path('src/View/Components/TestComponent.php'), $resolved);
```

Comment:
Shouldn't these be in `Http/Livewire`? I think this is calling the wrong command…